config.js

```diff
@@ -13,6 +13,11 @@ module.exports = {
       param: 'postcode'
     }
   },
+  upload: {
+    hostname: process.env.NODE_ENV === 'ci' || process.env.NODE_ENV === 'development' ?
+      `http://${process.env.LISTEN_HOST || '0.0.0.0'}:${process.env.PORT || 8080}/api/file-upload` :
+      'TODO-PROVIDE-URL'
```

Comment:
I know, but while there is no endpoint - @easternbloc is working on it - we need something, and a `TODO` that crashes and errors is better than something which fails silently.